zppy/e3sm_diags.py

```diff
@@ -34,8 +34,8 @@ def e3sm_diags(config: ConfigObj, script_dir: str, existing_bundles, job_ids_fil
         return existing_bundles
 
     # --- Generate and submit e3sm_diags scripts ---
-    dependencies: List[str] = []
     for c in tasks:
+        dependencies: List[str] = []
```

Comment:
The relevant changes from #623. We want to reset the dependencies list at the beginning of each task (i.e., nothing should be carried over).